test/integration/cli-image-create-kvm.test.js

```diff
@@ -94,10 +100,16 @@ test('triton image ...', testOpts, function (suite) {
     var markerFile = '/nodetritontest-was-here.txt';
     suite.test('  setup: triton create ... -n ' + ORIGIN_INST_ALIAS,
     function (t) {
+        if (!pkgId || !originImgNameOrId) {
+            t.comment('Cannot find pkg or origin image. Skip');
```

Comment:
Same here (move skip to be a prefix and capitalized).

Reply:
Done